### PR TITLE
[pebble][init] Add pebble service manager

### PR DESCRIPTION
**Pull request by arif-ali**
Closes: #3207
Resolves: SET-86

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [x] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

File: sos/policies/init_systems/pebble.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            list_cmd='services',
+            query_cmd='pebble services',
+            chroot=chroot
+        )
+        self.load_all_services()
+
+    def parse_query(self, output):
+        for line in output.splitlines()[1:]:
+            if line.split()[2] == 'active':
+                return line.split()[2]
```
Comment (pmoravec) [inline sos/policies/init_systems/pebble.py:30]: Why not simply `return 'active'`, then?

File: sos/policies/init_systems/pebble.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        for line in svcs:
+            try:
+                name = line.split()[0]
+                config = line.split()[1]
+                self.services[name] = {
+                    'name': name,
+                    'config': config,
+                }
+            except IndexError:
+                pass
```
Comment (TurboTurtle) [inline sos/policies/init_systems/pebble.py:44]: To keep CodeQL happy, let's add a comment here explaining either how we got to this exception (header/foooter output?) or some similar note to future-us.

**Review by TurboTurtle — COMMENTED** [review]
Looks good on its own (just the CodeQL note) - where do we anticipate this being used? Do we need to link up any existing policies to it now or is this prep work for something coming later?

File: sos/policies/init_systems/pebble.py (hunk 2 above)
Comment (arif-ali) [reply sos/policies/init_systems/pebble.py:44]: sure, it was a copy-paste from the systemd one. So, I think we ought to fix tat one too ;)

**Pull request closed without merging**